src/ast/nodes/ExportDefaultDeclaration.ts

```diff
@@ -48,10 +48,9 @@ export default class ExportDefaultDeclaration extends NodeBase {
 
 	initialise() {
 		this.included = false;
+		const declaration = this.declaration as FunctionDeclaration | ClassDeclaration;
```

Comment:
Added an intermediate variable because the type assertions seemed to confuse the compiler so it didn't to realize that the `id` property could never be `null`.